schema_salad/schema.py

```diff
@@ -177,7 +177,8 @@ def get_metaschema() -> Tuple[Names, List[Dict[str, str]], Loader]:
             },
             "typeDSL": saladp + "JsonldPredicate/typeDSL",
             "xsd": "http://www.w3.org/2001/XMLSchema#",
-        }
+        },
+        salad_version="v1.3",
```

Comment:
Ah, tricky. This seems to override the metaschema version in cwltool.

@tetron Any ideas? Perhaps we can't change TypeDSL; and make a TypeDSLv2 instead...

Reply:
Here are the difference between each parser/validator:

- cwltool does:
  - hard-coded metaschema v1.3 (in schema_salad/schema.py) -> Loader(v1.3) -> CWL v1.2 -> does not work!

- tested in schema_salad (runtime)
  - hard-coded metaschema v1.3 -> Loader(v1.3) -> metaschema.yml (v1.1) -> Loader(v1.1) -> CWL v1.2 -> it works!

- tested in schema_salad (code-generation)
  - hard-coded metaschema v1.3 -> Loader(v1.3) -> metaschema.yml (v1.1) -> Loader(v1.1) -> CWL v1.2 parser (generated from SALAD v1.1 spec) -> it works!

Reply:
False alarm with my local testing; nevermind!